### PR TITLE
Viewing and Updating configmaps and secrets

### PR DESCRIPTION
**Pull request by tt-le**
Add new section to view and edit configurations
issue: #34

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -324,6 +324,109 @@ X-App-Name: my-system
...
+Data
+====
+password:  6 bytes
+username:  3 bytes
+----
+
+You can quickly modify ConfigMaps by running the following command. This sets the key `name` to the value `new-name`. Note that this creates a new ConfigMaps and overwrites the old one. 
+[role=command]
+```
+kubectl create configmap sys-app-name --from-literal name=new-name --dry-run=client --save-config=true -o yaml | kubectl replace -f -
```
Comment (AustinSeto) [inline README.adoc:376]: Syntax error when running this command

```
Error: invalid argument "client" for "--dry-run" flag: strconv.ParseBool: parsing "client": invalid syntax
```

File: README.adoc [hunk 2]
```diff
@@ -324,6 +324,109 @@ X-App-Name: my-system
...
+[source, role="no_copy"]
+----
+X-App-Name: new-name
+----
+
+Let's change the app name again and redeploy to prepare for testing. 
+
+[role=command]
+```
+kubectl create configmap sys-app-name --from-literal name=my-system --dry-run=client --save-config=true -o yaml | kubectl replace -f -
```
Comment (AustinSeto) [inline README.adoc:426]: Same syntax error as previous

**Review by AustinSeto — CHANGES_REQUESTED** [review]
Got some syntax errors on the commands as I was running through the new section

File: README.adoc (hunk 1 above)
Comment (tt-le) [reply README.adoc:376]: What version of kubectl are you running

Comment (AustinSeto) [reply README.adoc:376]: My kubectl version is as follows:

```
Client Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.0", GitCommit:"2bd9643cee5b3b3a5ecbd3af49d09018f0773c77", GitTreeState:"clean", BuildDate:"2019-09-18T14:36:53Z", GoVersion:"go1.12.9", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.8", GitCommit:"9f2892aab98fe339f3bd70e3c470144299398ace", GitTreeState:"clean", BuildDate:"2020-08-13T16:04:18Z", GoVersion:"go1.13.15", Compiler:"gc", Platform:"linux/amd64"}
```

Comment (tt-le) [reply README.adoc:376]: I see, maybe its a version problem? Mine are as follows:
```
Client Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.4", GitCommit:"d360454c9bcd1634cf4cc52d1867af5491dc9c5f", GitTreeState:"clean", BuildDate:"2020-11-12T01:09:16Z", GoVersion:"go1.15.4", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.8", GitCommit:"9f2892aab98fe339f3bd70e3c470144299398ace", GitTreeState:"clean", BuildDate:"2020-08-13T16:04:18Z", GoVersion:"go1.13.15", Compiler:"gc", Platform:"linux/amd64"}
```
Could you try updating kubectl? `brew upgrade kubernetes-cli ` If you have `brew` installed

Comment (AustinSeto) [reply README.adoc:376]: Application works now, guess my version was just outdated

**Review by AustinSeto — APPROVED** [review]
Tested guide after upgrading my `kubectl`, all works well now

**Pull request closed without merging**